lib/ansible/cli/galaxy.py

```diff
@@ -197,7 +197,11 @@ def api(self):
 
 
 class GalaxyCLI(CLI):
-    '''command to manage Ansible roles in shared repositories, the default of which is Ansible Galaxy *https://galaxy.ansible.com*.'''
+    '''Command to manage Ansible roles and collections.
+
+       None of the CLI tools are designed to run concurrently with themselves,
+       you should use an external scheduler and/or locking to ensure there are no clashing operations.
```

Comment:
nit: I'd make these separate sentences. Short phrases are easier to consume for the reader.
```suggestion
       None of the CLI tools are designed to run concurrently with themselves.
       Use an external scheduler and/or locking to ensure there are no clashing operations.
```

Reply:
Fixed. I had just copy-pasted what the docs team had signed off on for `ansible-pull`.